docs/content/dataset_standards.md

```diff
@@ -177,10 +177,233 @@ The stepping data inside the episode group is divided into some required `datase
 </ul>
 </div>
 
-The required `datasets` found in the episode groups correspond to the data involved in every Gymnasium step call `obs, rew, terminated, truncated, info = env.step(action)`: `observations`, `actions`, `rewards`, `terminations`, and `truncations`. These datasets are `np.ndarray` and their shape is equal to:
+In the case where, for example, the observation space is a relatively complex `Dict` space with the following definition: 
```

Comment:
Minor but “for example” seems excessive tbh, simpler straightforward writing is usually better for documentation stuff (could say for example as the first two words of the sentence if it fits but in this case I think it should just be removed)

Reply:
removed